src/lib/data/tob.ts

```diff
@@ -484,29 +487,23 @@ export async function createTOBTeam({
 	let parsedTeam: ParsedTeamMember[] = [];
 
 	for (const u of team) {
-		const [kc, hardKC] =
-			kcOverride ?? (await Promise.all([u.getMinigameScore('tob'), u.getMinigameScore('tob_hard')]));
-
-		const [attempts, hardAttempts] =
-			attemptsOverride ??
-			(await Promise.all([
-				u.settings.get(UserSettings.Stats.TobAttempts),
-				u.settings.get(UserSettings.Stats.TobHardModeAttempts)
-			]));
-
 		let userPercentChange = 0;
 
 		// Reduce time for gear
-		const gearPerecents = calculateTOBUserGearPercents(u);
+		const gearPerecents = calculateTOBUserGearPercents(u.user);
```

Comment:
Is it possible to rewrite the calcTOBUserGear% function to use `u.gear` as argument to avoid unneeded db calls?

(I'm assuming the gear is actually passed, I didn't check the calling function to confirm)